src/CMakeLists.txt

```diff
@@ -24,6 +24,10 @@ if(Boost_UNIT_TEST_FRAMEWORK_FOUND)
     list(APPEND CUKE_SOURCES drivers/BoostDriver.cpp)
 endif()
 
+if(CGREEN_FOUND)
+    list(APPEND CUKE_SOURCES drivers/CgreenDriver.cpp)
```

Comment:
Just like is done for GTest (few lines above): you need to ensure Cgreen gets linked to, like this:
```cmake
    list(APPEND CUKE_DEP_LIBRARIES Cgreen::Cgreen)
```